paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -171,8 +172,15 @@ internal class PaymentOptionsActivityTest {
 
     @Test
     fun `ContinueButton should be hidden when returning to payment options`() {
+        val paymentMethodMetadata = PaymentMethodMetadataFactory.create(
```

Comment:
How come this change is necessary?

Reply:
I’m not quite sure. The test starts failing with an NPE deep inside Compose code, even though things work as expected on the client 🤔